src/model/ThermalStorageChilledWaterStratified.cpp

```diff
@@ -742,6 +742,25 @@ namespace model {
       }
     }
 
+    ComponentType ThermalStorageChilledWaterStratified_Impl::componentType() const {
+      return ComponentType::Cooling;
+    }
+
+    std::vector<FuelType> ThermalStorageChilledWaterStratified_Impl::coolingFuelTypes() const {
+      if (auto p_ = secondaryPlantLoop()) {
+        return p_->coolingFuelTypes();
+      }
+      return {};
+    }
```

Comment:
* [ ] TODO: should componentType be either 'None' or 'Both' instead? Should coolingFuelTypes be empty?

cf ThermalStorageIceDetailed too.

Reply:
Matching develop: https://github.com/NREL/OpenStudio/blob/4fd225f2ce4d7a5bc325337ff580942da38d0017/src/energyplus/ForwardTranslator/ForwardTranslatePlantEquipmentOperationSchemes.cpp#L523-L525